### PR TITLE
Update README_CN.md

### PR DESCRIPTION
**Pull request by EyreFree**
The README.md has changed a lot, so I updated the old README_CN.md to make it better.

### CODE REVIEW

File: README_CN.md [hunk 1]
```diff
@@ -61,13 +104,62 @@ Available commands:
...
+
+### Swift Version Support
+
+这里有一份 SwiftLint 版本和对应该 Swift 版本的对照表作为参考。
+
+| Swift 版本 | 最后一个 SwiftLint 支持版本 |
+| ------------- | -------------------------------- |
+| Swift 1.x     | SwiftLint 0.1.2                  |
+| Swift 2.x     | SwiftLint 0.18.1                 |
+| Swift 3.x     | Latest                           |
```
Comment (jpsim) [inline README_CN.md]: Shouldn't "Latest" be translated here and in the row below?

**Comment by jpsim** [comment]
Thanks for all this! Just have one thing that stuck out.

**Comment by EyreFree** [comment]
@jpsim Thanks for correcting me, it has been fixed.

**Comment by jpsim** [comment]
You've clearly put in a lot of work into this. Thank you!

**Comment by EyreFree** [comment]
Oh, It was my pleasure, I enjoyed this.

**Pull request merged**